src/lib/themes-test.js

```diff
@@ -300,6 +309,33 @@ describe("lib/themes", () => {
       ).to.be.true;
     });
 
+    it("should accept if only one has alpha set to opaque", () => {
+      expect(
+        subject(
+          { colors: { toolbar: { r: 1, g: 2, b: 3 } }, },
+          { colors: { toolbar: { r: 1, g: 2, b: 3, a: 1 } } }
+        )
+      ).to.be.true;
+    });
+
+    it("should accept if both alpha are set to opaque", () => {
+      expect(
+        subject(
+          { colors: { toolbar: { r: 1, g: 2, b: 3, a: 1 } }, },
+          { colors: { toolbar: { r: 1, g: 2, b: 3, a: 1 } } }
+        )
+      ).to.be.true;
+    });
+
+    it("should accept if both alpha are set to transparent", () => {
```

Comment:
Those are "good weather" tests. If you change the implementation below from `?? 1` to `|| 1`, will you see any test failure? From what I have seen in the tests, the answer appears to be no.

To get decent coverage, you should at the very leasy also be comparing `a: 0` with `a: 1` (expect false) and `a: 0` with `(no a)` (expect false).

And since the source code contains the (hardcoded) magic number `1`, it might be a good idea to compare these boundary values 0/1 with a different value in between (e.g. 0.5): compare `a: 0` with `a: 0.5`, and `a: 1` with `a: 0.5`.